tests/end2end/playwright/base-layers.spec.js

```diff
@@ -1,7 +1,13 @@
 // @ts-check
+import * as path from 'path';
+import * as fs from 'fs/promises'
+import { existsSync } from 'node:fs';
 import { test, expect } from '@playwright/test';
 import { gotoMap } from './globals';
 
+// To update MOCK files
```

Comment:
```suggestion
// To update OSM tiles in the mock directory
// IMPORTANT, this must not be set to `true` while commiting, on GitHub. Set to `false`.
```

Reply:
I have updated the comment. can you check it ?

Reply:
Ok for me